package.json

```diff
@@ -20,7 +20,8 @@
     "pretest": "npm run lint",
     "test": "karma start --single-run",
     "posttest": "istanbul report --root coverage text-summary && istanbul check-coverage --lines 95 --statements 95 --branches 95 --functions 95",
-    "test:watch": "karma start --auto-watch"
+    "test:watch": "karma start --auto-watch",
+    "report-coverage": "codecov"
```

Comment:
I suppose not. If we want to generate a report from dev we can just use the `codecov` command